### DISCUSSION TITLE
Update/Patch: Please Support EFS throughput type: "elastic"

### DISCUSSION THREAD

**Discussion by armenr** [terraform-linters/tflint-ruleset-aws#439] [Q&A] [answered]
```
3 issue(s) found:

Error: "elastic" is an invalid value as throughput_mode (aws_efs_file_system_invalid_throughput_mode)

  on efs.tf line 6:
   6:   throughput_mode                 = var.provisioned_throughput > 0 ? "provisioned" : var.throughput_mode
```

EFS "elastic" throughput was announced as Generally Available right at the end of November. See this link for reference:
https://aws.amazon.com/blogs/aws/new-announcing-amazon-efs-elastic-throughput/

The AWS provider supports it in terraform here:
https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/efs_file_system#throughput_mode

**Comment by wata727** [comment, marked as answer]
This is already supported in v0.21.0. Please try again.
https://github.com/terraform-linters/tflint-ruleset-aws/releases/tag/v0.21.0
